test/Ansible.common

```diff
@@ -9,17 +9,14 @@ $(ROLES_PATH) $(COLLECTIONS_PATH):
 
 .PHONY: ansible/dependencies
 ansible/dependencies: $(ROLES_PATH) $(COLLECTIONS_PATH)
-ifndef SSH_KEY
-	ansible-galaxy role install -r $(REQUIREMENTS_FILE) -p $(ROLES_PATH)
-	ansible-galaxy collection install -r $(REQUIREMENTS_FILE) -p $(COLLECTIONS_PATH)
-else ifeq ($(wildcard $(SSH_KEY)),)
-	ansible-galaxy role install -r $(REQUIREMENTS_FILE) -p $(ROLES_PATH)
-	ansible-galaxy collection install -r $(REQUIREMENTS_FILE) -p $(COLLECTIONS_PATH)
-else
+ifdef SSH_KEY
 	eval $$(ssh-agent -s) && \
 	ssh-add $(SSH_KEY) && \
```

Comment:
Wdyt of setting the "endif" here, so we don't replicate the ansible-galaxy command? (add the ssh key if defined, run the ansible-galaxy command in any case)

Reply:
Each line of a makefile uses a sub-shell, if we do it like this we will lose the access to the ssh-agent's socket env var 😞 that's why the command inside the SSH_KEY defined clause is actually a single line

Reply:
Unless we somehow populate it for later use inside the same recipe I'm not sure how to achieve it

Reply:
Make sense to me, thanks for the reply!